data/json/snippets/zombie_anatomy.json

```diff
@@ -171,6 +171,72 @@
       "You cut into its abdomen, and discover its utter lack of internal organs, the expected cavity replaced by nothing but more flesh and muscle."
     ]
   },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_grabber_general>",
+    "text": [
+      "The sheer remaining tension in each of the muscles of its arms and shoulders makes them unwind with a sonorous thwang each time you cut one of the supporting bone.",
+      "The tremendous growth of its upper body's muscle mass must have rendered the creature almost incapable of efficiently surveying its surroundings.",
+      "You realize the muscles of the zombie's arms are hollow, their marrow replaced by a thick oily black liquid. You wonder if the ridiculous muscle mass surrounding them is the other half of a strange hydraulic system.",
+      "You cut into the arms of the creature and realize that instead of what you would expect from what you know of normal human anatomy, its muscles wind around the bones in what looks like a tight spring..",
+      "While you are cutting into it, the creature suddenly moves and grabs one of your limbs. Despite considerable effort, you are unable to pry open the creature's iron grip and resign yourself to chop off the hand instead."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_dog_general>",
+    "text": [
+      "Opening its jaw wide, you notice the presence of multiple additional tongues deeper it in its throat.",
+      "You notice cysts in its fur, cutting into them causes an impressive amount of putrid black liquid to flow out.",
+      "The rear of this greying zombified dog shows strange reinforcements, as if a previous pathology was being remediated.",
+      "Inside its muzzle you notice strange glands and fibrous growths, almost as if the creature's sense of smell was in the process of being overhauled.",
+      "Lodged into the neck area of the creature and grown over you find a corroded tag reading the following <name> my best friend forever."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_electric_general>",
+    "text": [
+      "You find strange tubular glands across the creature contained alternating layers of two alien tissue types soaked in a black liquid, its smell putrid but saltier than usual for a zombie.",
+      "Uncomfortable static forces you to ground yourself multiple times during your work.",
+      "A roaming insect attracted by the carcass gets fried after landing on an organ glowing a faint blue, making you decide to avoid it with the utmost care.",
+      "The creature's heart seems to still be beating, but rather than pumping blood, it strangely reminds you of some sort of biological dynamo",
```

Comment:
```suggestion
      "The creature's heart seems to still be beating, but rather than pumping blood, it strangely reminds you of some sort of biological dynamo.",
```